--- soxr-0.1.3-Source/src/CMakeLists.txt.orig	2018-02-21 07:36:46.000000000 +0000
+++ soxr-0.1.3-Source/src/CMakeLists.txt	2020-12-04 21:07:53.956469225 +0000
@@ -101,7 +101,7 @@
   set (LSR_SO_VERSION 0.1.9)
   set (LSR_SO_VERSION_MAJOR 0)
   add_library (${LSR} ${LIB_TYPE} ${LSR})
-  target_link_libraries (${LSR} ${PROJECT_NAME})
+  target_link_libraries (${LSR} ${PROJECT_NAME} ${LIBM_LIBRARIES})
   set_target_properties (${LSR} PROPERTIES
     VERSION "${LSR_SO_VERSION}"
     SOVERSION ${LSR_SO_VERSION_MAJOR}
